src/QueryBuilder/typeQueries/stringQuery.ts

```
@@ -10,8 +10,10 @@ const escapeQueryString = (string: string) => {
 };
 
 // eslint-disable-next-line import/prefer-default-export
-export function stringQuery(compiled: CompiledSearchParam, value: string): any {
-    const fields = [compiled.path, `${compiled.path}.*`];
+export function stringQuery(compiled: CompiledSearchParam, value: string, modifier?: string): any {
+    const keywordSuffix = modifier === 'exact' ? '.keyword' : '';
+    const fields = [compiled.path + keywordSuffix, `${compiled.path}.*`];
+
```

We should throw an error if the modifier is something other than `exact`

---

Good catch, I updated this as well as all the other queries to utilize the `SUPPORTED_MODIFIERS` `const` to check if a modifier is valid.